build.sbt

```diff
@@ -101,6 +101,12 @@ lazy val httpMarshallersJava = project("akka-http-marshallers-java")
 lazy val httpJackson =
   httpMarshallersJavaSubproject("jackson")
 
+lazy val httpClusterManagement = project("akka-http-cluster-management")
```

Comment:
cc @patriknw @jrudolph how do you think we should mark this as experimental or somehow "fresh"...?